File: Programmers/최소직사각형.py
def solution(sizes):
    answer = 0
    max_width = 0
    max_height = 0
    for index in range(len(sizes)):
        if sizes[index][0] < sizes[index][1]:
            sizes[index] = [sizes[index][1], sizes[index][0]]
        
        if max_width < sizes[index][0]:
            max_width = sizes[index][0]
        if max_height < sizes[index][1]:
            max_height = sizes[index][1]
            
    return max_width*max_height

File: Programmers/최소직사각형.kt
class Solution {
    fun solution(sizes: Array<IntArray>): Int {
        var answer: Int = 0
        var max_width = 0
        var max_height = 0
        for(size in sizes){
            if (size[0] < size[1]){
                var temp = size[0]
                size[0]=size[1]
                size[1] = temp
            }
            if (max_width<size[0]){
                max_width =size[0]
            }
            if(max_height<size[1]){
                max_height = size[1]
            }
        }
       
        
        return max_width*max_height
    }
}